ftw/crawler/main.py

```diff
@@ -54,77 +55,102 @@ def get_indexing_time(url, indexed_docs, config):
 def crawl_and_index(tempdir, config, options):
     solr = SolrConnector(config.solr)
 
+    try:
+        from slack import SlackLogger
+        SlackLogger = SlackLogger(options.slacktoken)
+        HAS_SLACK = True
+    except ImportError:
```

Comment:
Use `pkg_resources.get_distribution` instead of catching `ImportError` - see  http://docs.plone.org/develop/styleguide/python.html#about-imports